lambda_function.py

```diff
@@ -59,8 +56,8 @@ def lambda_handler(event, lambda_context):
                     row["auth_type"] = message["authType"]
                     row["user_id"] = each["userID"]
                     row["user_data_validated"] = each["valid"]
-                    print(row)
-                    return insert_data(row)
+
+                    insert_data(row)
```

Comment:
Why is `return` removed? Does anything happen if we don't return from the main `lambda_handler` function?

Reply:
If you check the PR for multiple entries, I loop through the array. If I add a return it'll break the loop.